action.yml

```diff
@@ -0,0 +1,137 @@
+name: "GET Toolkit to lint and notify failures"
+description: "Lints using Dockle and ArgoCD diff and sends notification to slack"
+inputs:
+  registry:
+    description: "Registry's host"
+    required: false
+    default: "ghcr.io"
+  user:
+    description: "Registry's user"
+    required: false
+    default: ${{ github.actor }}
+  password:
+    description: "Registry user's password or token"
+    required: true
+  image:
+    description: "Docker image name with tag"
+    required: true
+  dockle:
+    description: "Enable dockle linting steps (Default: true)"
+    required: false
+    default: true
+  argocd:
+    description: "Enable argocd diff on PRs (Default: true)"
+    required: false
+    default: true
+  argocd-auth:
+    description: "ArgoCD Auth token for production"
+    required: false
+  argocd-auth-staging:
+    description: "ArgoCD Auth token for staging"
+    required: false
+  argocd-auth-testing:
+    description: "ArgoCD Auth token for testing"
+    required: false
+  argocd-server-testing:
+    description: "ArgoCD server URI for testing"
+    required: false
+  argocd-server-staging:
+    description: "ArgoCD server URI for staging"
+    required: false
+  argocd-server:
+    description: "ArgoCD server URI for production"
+    required: false
+  slack-webhook:
+    description: "Slack Webhook for notifications"
+    required: true
+  deploy-fail-notification:
+    description: "Send notification to Slack for deploy failure"
+    required: false
+    default: false
+
+runs:
+  using: "composite"
+  steps:
```

Comment:
This should be multiple steps that can be parapele executed.

Reply:
How tho? Composite actions don't work like workflows and can't have multiple jobs.